cmd/catalog/main.go

```diff
@@ -69,6 +69,8 @@ var (
 
 	profiling = flag.Bool(
 		"profiling", false, "serve profiling data (on port 8080)")
+
+	installPlanTimeout = flag.Duration("install-plan-retry-timeout", 1*time.Minute, "time since first attempt at which plan execution errors are considered fatal")
```

Comment:
should the default live as an exported constant somewhere like the default resync interval of 15 minutes in the `queueinformer` package?

Reply:
I don't think any calling code needs to depend on it. IMO the fact that there is a default is a concern of the CLI at this point.